lib/gui/tool-runner/index.ts

```diff
@@ -173,7 +173,11 @@ export class ToolRunner {
 
         return Promise.all(tests.map(async (test): Promise<TestBranch> => {
             const updateResult = this._createTestplaneTestResult(test);
-            const currentResult = formatTestResult(updateResult, UPDATED, test.attempt);
+            const latestAttempt = reportBuilder.getLatestAttempt({
+                fullName: updateResult.fullTitle(),
+                browserId: updateResult.browserId
+            });
```

Comment:
`updateResult` has `TestplaneTestResult` type.
Since `reportBuilder` is not currently tied to `Testplane`, im not bringing `Testplane*` to `reportBuilder` either.